_assignments/mp3.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: assignment
+title: "MP3: Perception in VR"
+index: 30
+due: "Mar. 13, 2019, @ 11:59 PM"
+material:
+  -
+    name: MP3 Unity Package
+    url: /mps/MP3.unitypackage
+rubric:
+  -
+    name: Position Tracking
+    points: 20
```

Comment:
Recommend to change the breakdown from 20 for position and 40 for rotation, to
15 for position, 25 for rotation, 25 for both position and rotation simultaneously. note the extra 5 that we have b/c of below